### PR TITLE
Automated Test - COMMERCE-6027 | master

### PR DESCRIPTION
**Pull request by evanilson-santana**

### CODE REVIEW

**Comment by evanilson-santana** [comment]
https://issues.liferay.com/browse/COMMERCE-6027 , cc: @Pietro69

**Comment by pedro-michel** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase [hunk 1]
```diff
@@ -147,12 +147,41 @@ definition {
...
+
+		MenuItem.click(menuItem = "Edit");
+
+		SelectFrame.selectFrameNoLoading(locator1 = "CommerceEntry#IFRAME_SIDE_PANEL");
+
+		Type(
+			locator1 = "TextInput#TITLE",
+			value1 = "Commerce_Black_Edit");
+
+		Button.clickSaveAsDraft();
```
Comment (Pietro69) [inline modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CPCommerceProductMedia.testcase:181]: I'm sorry I wrote wrong in the ticket when I said save I meant to click on the publish button

**Review by Pietro69 — CHANGES_REQUESTED** [review]

**Comment by Pietro69** [comment]
Please add the `property test.name.skip.portal.instance = "CPCommerceProductMedia#EditAnImage"`

**Comment by evanilson-santana** [comment]
@Pietro69 Fixed

**Comment by pedro-michel** [comment]
ci:test:sf

**Comment by iovaneale** [comment]
ci:test:commerce-functional

**Pull request closed without merging**